libcontainer/seccomp/config.go

```diff
@@ -74,6 +74,7 @@ var archs = map[string]string{
 	"SCMP_ARCH_RISCV64":     "riscv64",
 	"SCMP_ARCH_S390":        "s390",
 	"SCMP_ARCH_S390X":       "s390x",
+	"SCMP_ARCH_LOONGARCH64": "loong64",
```

Comment:
This should be `loongarch64` as this is how it's called in libseccomp and libseccomp-golang.

Reply:
Nevermind; libseccomp-golang v0.11.1 now supports "loong64".